app/components/home.js

```diff
@@ -0,0 +1,125 @@
+import React, { Component } from 'react';
+import {
+    Alert,
+    StyleSheet,
+    FlatList,
+    View,
+    Text,
+    ActivityIndicator, TouchableHighlight, ActionSheetIOS
+} from 'react-native';
+
+import {Actions} from 'react-native-router-flux'
+
+
+class Home extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {};
+    }
+
+    // Dummy function for button presses
+    pressButton(label){
+        Alert.alert(label);
+    }
+
+    // Displays animation if loading, otherwise displays a popup indicating the
+    // TouchableHighlight pressed
+
+    render() {
+
+        if (this.props.loading) {
+            return (
+                <View style={styles.activityIndicatorContainer}>
+                    <ActivityIndicator animating={true}/>
+                </View>
+            );
+        } else {
+            return (
+                <View style={styles.container}>
+            
+                    <TouchableHighlight onPress={() => {this.pressButton("Inbox")}}>
```

Comment:
I feel like going forward we will should just link all components to the home screen for sanity purposes that way we don't have to keep reorganizing what component is rendered when the app first opens. But that's something for next sprint and not this one since we pretty much don't have any other components.

Reply:
Good idea, I'll add that as a task for next sprint.